packages/react-ui/storybook/stories/widgetsUI/ComparativeCategoryWidgetUI.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
Stories file is obviously very useful for debugging, thx for doing it.

But something we're missing here, and in general in comparative is the unit testing side. Same as we have /_tests__/widgets/CategoryWidgetUI.test.js we should have corresponding unit tests for each comparative widget. 

That's gonna allow us to maintain it better in the future (or maybe to refactor it to share some pieces...). Would you please?
Would you please

Reply:
Allright, I will be looking at existing test files and try to create similar ones for comparative widgets

Reply:
And this one will be a task for tomorrow